backend/src/main/java/com/bakdata/conquery/apiv1/ApiV1.java

```diff
@@ -43,6 +43,7 @@ public void registerResources(ManagerNode manager) {
 			protected void configure() {
 				bind(manager.getConfig()).to(ConqueryConfig.class);
 
+				// TODO some of these have been bind previously -> clean that up

```

Comment:
Noch offen für den PR?

Reply:
Das würde ich in dem hier angehen: https://github.com/bakdata/conquery/pull/2484